cmd/dumpling/main.go

```diff
@@ -119,6 +120,7 @@ func main() {
 	pflag.StringVar(&keyPath, "key", "", "The path name to the client private key file for TLS connection")
 	pflag.StringVar(&csvSeparator, "csv-separator", ",", "The separator for csv files, default ','")
 	pflag.StringVar(&csvDelimiter, "csv-delimiter", "\"", "The delimiter for values in csv files, default '\"'")
+	pflag.StringVar(&outputFilenameFormat, "output-filename-format", "{{.DB}}.{{.Table}}.{{.Index}}", "The output filename format, default '{{.DB}}.{{.Table}}.{{.Index}}'")
```

Comment:
```suggestion
	pflag.StringVar(&outputFilenameFormat, "output-filename-template", "{{.DB}}.{{.Table}}.{{.Index}}", "The output filename template (without file extension), default '{{.DB}}.{{.Table}}.{{.Index}}'")
```

Reply:
addressed in https://github.com/pingcap/dumpling/pull/120/commits/50d18ad53396b887b1cbf08309eaeb153dd48638